### ISSUE TITLE
Design: Страница музыканта

### ISSUE THREAD

**Issue by rogulenkoko** [rogulenkoko/FunkMap-Front#3]
Страница должна содержать следующую информацию
1. Фотография
2. Имя, возраст, пол
3. Описание
4. Видео из YouTube (необязательно)
5. Ссылки на вк, ютуб, фейсбук

Страница будет появляется при нажатии на маркер на карте. Закрывает боковую панель.
Имеет цвет отличный от цвета страницы музыканта, пренадлежащего авторизованному пользователю. В центре незакрытой части карты появляется выделенный маркер из которого понятно, на каком инструменте он играет и его локация.

**Comment by VitalyGrigorjev** [comment]
Я считаю, что стоит так же указывать стаж игры, инструмент, возможно стиль? нужно подумать над этим

**Comment by VitalyGrigorjev** [comment]
Исходник и превью. 

https://drive.google.com/open?id=0B3VWeKEo7dyyNV9lcEpCN3oyTlk

**Comment by rogulenkoko** [comment]
Да, про стаж я забыл как-то. А инструмент будет понятен по метке на карте

**Comment by VitalyGrigorjev** [comment]
Обновл исходник и превью
https://drive.google.com/open?id=0B3VWeKEo7dyyTlA0X1lUWVQ0XzA
